api/src/main/java/org/ehcache/Maintainable.java

```diff
@@ -34,4 +34,9 @@
    * @throws java.lang.IllegalStateException if data structures don't exist
    */
   void destroy();
+
+  /**
+   * Exit maintenance mode
+   */
+  void exit();
```

Comment:
I thought Maintainable was closable?  Appears it isn't, but it probably should be.